src/main/steps/applicant1/how-to-finalise/template.njk

```diff
@@ -0,0 +1,8 @@
+{% extends "common/page.njk" %}
+
+{% block page_content %}
```

Comment:
this looks like it doesn't need a custom template, because the content is the same as the standard template (src/main/steps/common/template.njk)

Reply:
fixed